src/panels/ShareModelPanel/ShareModel.tsx

```diff
@@ -39,28 +124,232 @@ export default function ShareModel() {
             </div>
           }
         />
-        <div className="p-panel__content">
-          <div>
+        {errorMsg && (
+          <div className="p-notification--negative">
+            <p className="p-notification__response" role="status">
+              <span className="p-notification__status">Error:</span>
+              {errorMsg}
+            </p>
+            <button
+              className="p-icon--close"
+              aria-label="Close notification"
+              aria-controls="notification"
+              onClick={() => {
+                setErrorMsg(null);
+              }}
+            >
+              Close
+            </button>
+          </div>
+        )}
+        <div className="p-panel__content aside-split-wrapper">
+          <div className="aside-split-col">
             <h5>Sharing with:</h5>
             {users?.map((userObj: User) => {
+              const userName = userObj["user"];
+              const lastConnected = userObj["last-connection"];
+
               return (
                 <div className="share-model__card" key={userObj.user}>
                   <div className="share-model__card-title">
-                    <strong>{userObj["user"]}</strong>
+                    <strong>{userName}</strong>
                     <span className="secondary">
-                      {isOwner(userObj["user"]) && "Owner"}
+                      {isOwner(userName) ? (
+                        "Owner"
+                      ) : (
+                        <i
+                          className="p-icon--delete"
+                          onClick={() => handleRemoveUser(userName)}
+                          onKeyPress={() => handleRemoveUser(userName)}
+                          role="button"
+                          tabIndex={0}
+                        >
+                          Remove user
+                        </i>
+                      )}
                     </span>
                   </div>
-                  <p className="supplementary">
+                  <div className="supplementary">
                     Last connected:{" "}
-                    {userObj["last-connection"]
-                      ? formatFriendlyDateToNow(userObj["last-connection"])
+                    {lastConnected
+                      ? formatFriendlyDateToNow(lastConnected)
                       : `Never connected`}
-                  </p>
+                    <Formik
+                      initialValues={{
+                        selectAll: false,
+                        selectedUnits: [],
```

Comment:
I'm not sure what any of these props are for?

Reply:
Ahh.. these are a copy over from the actions forms which I forgot to delete. This also explains why I thought the FormikFormData component used in Actions was leaking into this view.